File: CS/WinWebSolution.Module.Win/WinSchedulerFilterResourcesListViewController.cs
using DevExpress.ExpressApp;
using DevExpress.Data.Filtering;
using DevExpress.Xpo;

namespace WinWebSolution.Module {
    public class WinSchedulerFilterResourcesListViewController : SchedulerFilterResourcesListViewControllerBase {
        protected override void FilterResourcesDataSource(object resourcesDataSource, CriteriaOperator criteria) {
            if (resourcesDataSource is XPCollection) {
                ((XPCollection)resourcesDataSource).Criteria = criteria;
            }
        }
    }
}

File: VB/WinWebSolution.Module.Win/WinSchedulerFilterResourcesListViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.Data.Filtering
Imports DevExpress.Xpo

Namespace WinWebSolution.Module
	Public Class WinSchedulerFilterResourcesListViewController
		Inherits SchedulerFilterResourcesListViewControllerBase
		Protected Overrides Sub FilterResourcesDataSource(ByVal resourcesDataSource As Object, ByVal criteria As CriteriaOperator)
			If TypeOf resourcesDataSource Is XPCollection Then
				CType(resourcesDataSource, XPCollection).Criteria = criteria
			End If
		End Sub
	End Class
End Namespace